### PR TITLE
[CPU] Performance fixes

### PR DESCRIPTION
**Pull request by mandrono**
Issue: 55340, 55352

### CODE REVIEW

**Comment by mandrono** [comment]
@dmitry-gorokhov take a look please

File: inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_bias.cpp [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace SubgraphTestsDefinitions {
+
+using ConvBiasTestParams = std::tuple<nodeType,
+                                      SizeVector,  // input shape
+                                      SizeVector>; // kernel size
+
+class ConvBiasTest : public testing::WithParamInterface<ConvBiasTestParams>, virtual public LayerTestsUtils::LayerTestsCommon {
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_bias.cpp:19]: CPU specific Convolution test supports test cases with post ops. Why we need separate one?

File: inference-engine/src/mkldnn_plugin/mkldnn_node.h [hunk 2]
```diff
@@ -651,6 +589,7 @@ class MKLDNNNode {
 protected:
     bool canBePerformedAsScaleShift(const MKLDNNNode *parentNode = nullptr) const;
     bool canFuseSimpleOperation(const MKLDNNNodePtr& node) const;
+    void fillScalesAndShifts(const MKLDNNNode *parentNode, std::vector<float> &scales, std::vector<float> &shifts, const int align = -1);
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/mkldnn_node.h:592]: Not sure I like the implementation. For now we can keep as is, but in general I would prefer it to be placed outside of the node API.

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp [hunk 3]
```diff
@@ -1524,64 +1524,67 @@ void MKLDNNGraphOptimizer::FuseClampAndFakeQuantize(MKLDNNGraph &graph) {
...
+            return 0;
+        } else if (node->getParentEdgeAt(1)->getParent()->isConstant() && node->getParentEdgeAt(1)->getParent()->getType() == Input) {
+           return 1;
+        } else {
+            return -1;
+        }
+    };
+
+    auto isSutableScaleShiftNode = [getConstPort](MKLDNNNodePtr node) {
+        if (one_of(node->getAlgorithm(), EltwiseAdd, EltwiseMultiply, EltwiseMulAdd)) {
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp:1541]: What about Sub or Div?

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp [hunk 4]
```diff
@@ -1524,64 +1524,67 @@ void MKLDNNGraphOptimizer::FuseClampAndFakeQuantize(MKLDNNGraph &graph) {
...
+            MKLDNNNode *parent = nullptr;
+            if (node->getAlgorithm() != EltwiseMulAdd) {
+                const auto constPort = getConstPort(node);
+                if (constPort == -1) {
+                    return false;
+                }
+                parent = node->getParentEdgeAt(1 - constPort)->getParent().get();
+            }
+            return node->getType() == Eltwise && node->getChildEdges().size() == 1 && node->canBePerformedAsScaleShift(parent);
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp:1550]: Why canBePerformedAsScaleShift is called for parent of Eltwise operation?

File: inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_bias.cpp (hunk 1 above)
Comment (mandrono) [reply inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_bias.cpp]: existing tests was updated

File: inference-engine/src/mkldnn_plugin/mkldnn_node.h (hunk 2 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_node.h:593]: todo was added

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp (hunk 3 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp]: sub and div was added, thanks!

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp (hunk 4 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp:1549]: ```canBePerformedAsScaleShift``` called for ScaleShift candidate, ```parent``` used inside ```canBePerformedAsScaleShift``` to separate data input and const nodes

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**